BlocksForAll/Assets.xcassets/Orange.imageset/Contents.json

```diff
@@ -2,20 +2,14 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "Orange.pdf",
-      "scale" : "1x"
-    },
-    {
-      "idiom" : "universal",
-      "scale" : "2x"
-    },
-    {
-      "idiom" : "universal",
-      "scale" : "3x"
+      "filename" : "Orange.pdf"
     }
   ],
   "info" : {
     "version" : 1,
     "author" : "xcode"
```

Comment:
should be:
19-20
11-13
  },
  "properties" : {
    "preserves-vector-representation" : true